content/docs/actors/development/actor_config.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Actor config
+description: Learn how to write the main actor config in the `.actor/actor.json` file.
+paths:
+    - actor/development/actor-config
+    - actors/development/actor-config
+---
+
+# [](#actor-config) Actor config
+
+The main actor config should live in the `.actor/actor.json` file in the actor's root directory. This file associates your local development project with an actor on the Apify platform. It contains information such as actor name, version, build tag and environment variables. Make sure you commit this file to the Git repository.
+
+For example, `.actor/actor.json` file can look as follows:
+
+
+```json
+{
+  "actorSpecification": 1, // always 1
+  "name": "name-of-my-scraper",
+  "version": "0.0",
+  "buildTag": "latest",
+  "environmentVariables": {
+      "MYSQL_USER": "my_username",
+      "MYSQL_PASSWORD": "@mySecretPassword"
+  },
+  "dockerfile": "./Dockerfile",
+  "readme": "./ACTOR.md",
+  "input": "./input_schema.json",
+  "storages": {
+    "dataset": "./dataset_schema.json",
+  }
+}
+```
+
+Note that actor `name`, `version` and `buildTag` are for informative purposes only and are so far not enforced on the platform. You will still need to properly name your actor during its creation, whether you create it directly on the platform or you push it using Apify CLI.
+
+**`Dockerfile` field**\
```

Comment:
BTW, should these be sub-headings (h2)?

Reply:
I think it looks pretty ugly, they are not that relevant, I'd use just bold